docs/custom_domains.rst

```diff
@@ -133,41 +133,52 @@ You will need to update your CAA records to allow us to issue the certificate.
    to remove the domain name from their DNS Zone in order for your domain name to work with Read the Docs,
    else it will always redirect to GitBook.
 
-Proxy SSL
----------
-
-.. warning::
-
-   This option is deprecated,
-   we already issue SSL certificates for all domains.
-
-If you would prefer to do your own SSL termination
-on a server you own and control,
-you can do that although the setup is a bit more complex.
-
-Broadly, the steps are:
-
-* Have a server listening on 443 that you control
-* Procure an SSL certificate for your domain and provision it
-  and the private key on your server.
-* Add a domain that you wish to point at Read the Docs
-* Enable proxying to us, with a custom ``X-RTD-SLUG`` header
-
-An example nginx configuration for pip would look like:
-
-.. code-block:: nginx
-   :emphasize-lines: 9
-
-    server {
-        server_name pip.pypa.io;
-        location / {
-            proxy_pass https://pip.readthedocs.io:443;
-            proxy_set_header Host $http_host;
-            proxy_set_header X-Forwarded-Proto https;
-            proxy_set_header X-Real-IP $remote_addr;
-            proxy_set_header X-Scheme $scheme;
-            proxy_set_header X-RTD-SLUG pip;
-            proxy_connect_timeout 10s;
-            proxy_read_timeout 20s;
-        }
-    }
+Canonical URLs
+--------------
+
+Canonical URLs allow people to have consistent page URLs for domains.
+This is mainly useful for search engines,
+so that they can send people to the correct page.
+
+Read the Docs uses these in two ways:
+
+* We point all versions of your docs at the "latest" version as canonical
+* We point at the user specified canonical URL, generally a custom domain for your docs.
+
+Example
++++++++
```

Comment:
This is another thing we should have tin the style guide. I usually use a different 3rd-level header char (`~`), but it isn't standardized.

Reply:
Added that to https://github.com/readthedocs/readthedocs.org/issues/8108